lib/extensions/models_extensions.dart

```diff
@@ -26,8 +24,6 @@ extension NavigatorEntriesExt on BuiltList<PageData> {
         key: ValueKey(AuthOrMain), child: AuthOrMain()),
     EmailAuthPageData(): MaterialPage<EmailAuthOptionsPage>(
         key: ValueKey(EmailAuthOptionsPage), child: EmailAuthOptionsPage()),
-    OtherAuthOptionsPageData(): MaterialPage<OtherAuthOptionsPage>(
```

Comment:
removed as we don't have a separate page any more

Reply:
noice